sdlf-team/scripts/template-team-repos.yaml

```diff
@@ -26,7 +26,8 @@ Mappings:
       branch: master
 
```

Comment:
Since we have added a `pSNSNotificationsEmail` parameter in the above `bootstram_team.sh` script, I believe this template should also include it
```
  pSNSNotificationsEmail:
    Description: List of SNS Notification Email Endpoints 
    Type: CommaDelimitedList
```

Reply:
Adding it. Thanks Abdel